docs/Getting-Started/ReactRouter3.md

```diff
@@ -71,6 +71,8 @@ const userIsNotAuthenticated = connectedRouterRedirect({
 <Route path="login" component={userIsNotAuthenticated(Login)}/>
 ```
 
+The `locationHelper` requre `location` object in props. Therefore if component is not directly route component then `withRouter` must be applied before, for example like this `withRouter(userIsNotAuthenticated(Login))`.
```

Comment:
I see what you are saying, but the wording is a little awkward here. How about:

The `locationHelper` requires the `location` object in props. If the component is not rendered as part of a route component then you must use the `withRouter` HOC from react-router:
`withRouter(userIsNotAuthenticated(Login))`

Reply:
done at #230